src/main/kotlin/net/atos/zac/smartdocuments/rest/RestMappedSmartDocumentsTemplateGroup.kt

```diff
@@ -90,10 +90,10 @@ private fun convertTemplateGroupToModel(
     val jpaGroup = createModelTemplateGroup(group, parent, zaakafhandelParameterId)
 
     jpaGroup.templates = group.templates?.map {
-        createModelTemplate(it as RestMappedSmartDocumentsTemplate, jpaGroup, zaakafhandelParameterId)
+        createModelTemplate(it, jpaGroup, zaakafhandelParameterId)
     }?.ifEmpty { null }?.toMutableSet()
```

Comment:
```suggestion
    }?.toMutableSet() ?: null
```
This is the same, but why not return an empty set? Nullable values, particular collections, are a pain IMHO.

Reply:
Will get rid of `ifEmpty`